Catch (and display the type of) the exception that occurs when the following code is executed:
int[] m = new int[2];
m[8] = 5;

Requirements:
The program should display a message on the screen.
The program must have a try-catch block.
The program must catch a specific type of exception, not all possible exceptions (i.e. not Exception).
The displayed message must contain the type of exception that occurred.
Don't delete the existing code in the main method.






package com.codegym.task.task09.task0909;

/* 
Exception when working with arrays

*/

public class Solution {
    public static void main(String[] args) throws Exception {
       try
       {
        int[] m = new int[2];
        m[8] = 5;
       }

        catch(ArrayIndexOutOfBoundsException e){
            System.out.println("ArrayIndexOutOfBoundsException is caught");
        }
    }
}
